src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/test/Configuration/WattTimeClientConfigurationTests.cs

```diff
@@ -8,6 +8,7 @@ namespace CarbonAware.DataSources.WattTime.Tests;
 public class WattTimeClientConfigurationTests
 {
     [TestCase("testuser", "12345", "http://example.com", TestName = "Validate does not throw: username; password; url")]
+    [TestCase("testuser", "pass£word", "http://example.com", TestName = "Validate throws: username; non-ASCII password; url")]
```

Comment:
TestName and the test method name is inconsistent. Should be TestName="Validate _does not_ throw...